native/src/components/TtsContainer.tsx

```diff
@@ -110,7 +131,7 @@ const TtsContainer = ({ children }: TtsContainerProps): ReactElement => {
     async (index = sentenceIndex) => {
       const safeIndex = Math.max(0, index)
       const sentence = sentences[safeIndex]
-      if (sentence) {
+      if (sentence && enabled) {
```

Comment:
🔧 This should not be needed if other suggestions are applied.

Reply:
This is needed even with applied changes:
`if (sentence && isLanguageSupported) {` 

Changing language while the player is still open will let the user play even if not supported so I added this flag.

Reply:
The player should be closed if the new language is not supported. Otherwise there is no feedback to the user why the tts feature is disabled.